.gitignore

```diff
@@ -3,6 +3,7 @@
 /bazel-*
 .DS_Store
 /.vscode/*
+/.idea/*
```

Comment:
https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Hey if VSCode gets to be in there ;)

Reply:
I think that one is my fault, and now :shamecube: